.editorconfig

```diff
@@ -0,0 +1,16 @@
+[*]
+charset = utf-8
+end_of_line = lf
+ij_continuation_indent_size = 8
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+max_line_length = 120
+
+[pom.xml]
+max_line_length = 140
+
+[*.java]
+ij_java_class_count_to_use_import_on_demand = 999
+ij_java_imports_layout = $*, |, java.**, |, javax.**, |, org.**, |, net.**, |, com.**, |, *
```

Comment:
FYI: this special order is kind of tricky I was glad to found it online.



references:
https://github.com/checkstyle/checkstyle/pull/16574
https://github.com/checkstyle/checkstyle/pull/16598